app/Platform/Components/GameTitle.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Components;
+
+use Illuminate\Contracts\View\View;
+use Illuminate\View\Component;
+
+class GameTitle extends Component
+{
+    public string $rawTitle = '';
+    public bool $isDisplayingTags = true;
+
+    public function __construct(string $rawTitle, bool $isDisplayingTags = true)
```

Comment:
`isDisplayingTags` (interrogative) is more aligned with querying properties. Consider using an imperative (`showTags` ) for setting the value.

Reply:
No strong feeling on this so I'm happy to defer to your judgment. Changed in https://github.com/RetroAchievements/RAWeb/pull/1882/commits/5eabb2ed92294a86cbd4ef500d119b9de75f66b7.